Projects/hCoreUI/Sources/InfoView/InfoView.swift

```diff
@@ -82,16 +82,19 @@ public struct InfoViewHolder: View {
 public struct InfoView: View {
     let title: String
     let description: String
+    let markdownOnTap: (() -> Void)?
```

Comment:
Can we just rename this to:     let onUrlClicked: ((_ url: URL) -> Void)?
Maybe we want to handle url in the future